update-bot/wlmbots/checker_bot.py

```diff
@@ -94,20 +106,48 @@ def generate_category_result_table(self, results):
 
     def generate_config_table(self):
         line_fmt = "|-\n|[[Vorlage:{}|{}]]\n|{}\n|{}\n"
-        text = '{| class="wikitable"\n|-\n!Vorlage!!Bezeichner ID!!Format ID\n'
+        text = "== Unterstützte Vorlagen ==\nDie Seiten wurden mit folgenden Vorlagen und Einstellungen geprüft:\n"
+        text += '{| class="wikitable"\n|-\n!Vorlage!!Bezeichner ID!!Format ID\n'
         for template_name, config in sorted(self.checker.config.items()):
             text += line_fmt.format(template_name, template_name, config["id"], config["id_check_description"])
         text += "|}\n\n"
         return text
 
+    def save_wikipage(self, page_text, page_name, summary="Bot: Update der Ergebnisliste"):
+        try:
+            article = pywikibot.Page(self.site, page_name)
+            if not article.exists():
+                article.text = page_text
+                article.save(summary=summary)
+                return
+            old_text = article.get()
+            if old_text != page_text:
+                article.text = page_text
+                article.save(summary=summary)
```

Comment:
This looks like it can be shortened to

    if not article.exists() or article.get() != page_text:
        article.text = page_text
        article.save(summary=summary)
    else:
        ...